applications/StructuralMechanicsApplication/custom_response_functions/adjoint_conditions/adjoint_semi_analytic_point_load_condition.cpp

```diff
@@ -133,12 +133,11 @@ namespace Kratos
     {
```

Comment:
When CalculateSensitivityMatrix of the adjoint point load condition is called for a double variable an error is thrown. Consequently we have return also there a ZeroMatrix instead.

Reply:
fixed. thx for spotting :+1: